modules/rosa-create-objects.adoc

```diff
@@ -861,6 +861,73 @@ Add a machine pool with tags to a cluster.
 $ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --tags='foo bar,bar baz' --name=mp-1
 ----
 
+[id="rosa-create-network_{context}"]
+== create network
+
+Create a network that creates any necessary AWS resources through AWS CloudFormation templates. This helper command is intended to help create and configure a VPC for use with ROSA with HCP.
+
+[NOTE]
+====
+For custom or advanced configuration, it is highly recommended to use the AWS CLI directly using the `aws cloudformation` command.
+====
+
+[IMPORTANT]
+====
+Running this command creates resources within your AWS account.
+====
+
+.Syntax
+[source,terminal]
+----
+$ rosa create network [flags]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|<template-name>
+|Allows you to use a custom template. Templates must be in the template folder, structured as _templates/<template-name>/cloudformation.yaml_. If no template name is provided, command uses the default template.
+
+|===
+
+.Default template YAML
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/rosa/refs/heads/master/cmd/create/network/templates/rosa-quickstart-default-vpc/cloudformation.yaml[]
+----
+
+.Flags
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--template-dir
+|Allows you to specify the path to the template directory. Overrides the OCM_TEMPLATE_DIR environment variable. Required if not running the command inside the template directory.
+
+|--param Name
+|Define the name of your network. A required parameter when using a custom template file.
+
+|--param Region
+|Define the region of your network. A required parameter when using a custom template file.
+
+|--param <various>
+|Available parameters depend on the template. Use `--help` when in the template directory to find available parameters.
+
+|--mode=manual
+|Provides AWS commands to create the network stack.
+
+|===
+
+.Example
+Create a basic network with regular arguments and flags.
+
+[source,terminal]
+----
+$ rosa create network --param Tags=key1=value1,key2=value2 --param Name=example-stack --param Region=us-west-2
```

Comment:
```suggestion
$ rosa create network rosa-quickstart-default-vpc --param Tags=key1=value1,key2=value2 --param Name=example-stack --param Region=us-west-2
```

Reply:
It will be helpful if we provide template name as well in example usage

Reply:
+1

Reply:
Added.